src/deploy-airkeeper.ts

```diff
@@ -0,0 +1,65 @@
+import { join } from 'path';
+import { PromptObject } from 'prompts';
+import { OperationsRepository } from './types';
+import { runAndHandleErrors, runShellCommand } from './utils/cli';
+import { promptQuestions } from './utils/prompts';
+import { readOperationsRepository } from './utils/read-operations';
+
+const questions = (operationsRepository: OperationsRepository): PromptObject[] => {
+  return [
+    {
+      type: 'autocomplete',
+      name: 'name',
+      message: 'Which API Integration do you want to deploy?',
+      choices: Object.keys(operationsRepository.apis).map((api) => ({ title: api, value: api })),
+    },
+    {
+      type: 'autocomplete',
+      name: 'deployment',
+      message: (prev) => `Which deployment of ${prev} do you want to deploy?`,
+      choices: (prev) =>
+        Object.keys(operationsRepository.apis[prev].deployments).map((deployment) => ({
+          title: deployment,
+          value: deployment,
+        })),
+    },
+  ];
+};
+
+const main = async () => {
+  const airkeeperVersion = require('@api3/airkeeper/package.json').version;
+  const operationsRepository = readOperationsRepository();
+  const response = await promptQuestions(questions(operationsRepository));
+
+  const deploymentDirectory = join(__dirname, '..', 'data', 'apis', response.name, 'deployments', response.deployment);
+  const awsSecretsFilePath = join(deploymentDirectory, 'aws.env');
+
+  const config = operationsRepository.apis[response.name].deployments[response.deployment].config;
+  const stage = config.nodeSettings.stage;
+  const region =
+    config.nodeSettings.cloudProvider.type === 'aws' ? config.nodeSettings.cloudProvider.region : 'us-east-1';
+
+  const airkeeperDeployCommand = [
+    `docker run -it --rm`,
+    `--env-file ${awsSecretsFilePath}`,
+    `-v ${deploymentDirectory}:/app/config`,
+    `api3/airkeeper:${airkeeperVersion} deploy --stage ${stage} --region ${region}}`,
+  ]
+    .filter(Boolean)
+    .join(' ');
+
+  console.log(`⏳ - Deploying Airkeeper`);
+
+  runShellCommand(airkeeperDeployCommand);
+
+  console.log(
+    [
+      `🗽 - Airkeeper has been deployed`,
+      `⏩ - if you also deployed Airnode, please forward the "receipt.json" to the API3 team.`,
+      `The "receipt.json" contains sensitive information and should not be shared or made public.`,
```

Comment:
Yes Airkeeper doesn't contain the "receipt.json", I included this in here because the flow is usually deploy airnode then deploy airkeeper and it would good to remind the user to forward the receipt after airkeeper is deployed but I can remove it if it seems unnecessary.